src/config.rs

```diff
@@ -353,6 +347,34 @@ impl GitHub {
     }
 }
 
+/// Take a quoted string from the config file and unescape it (i.e. strip the start and end quote
+/// (") characters and process any escape characters in the string.)
+fn unescape_str(us: &str) -> String {
+    // The regex in config.l should have guaranteed that strings start and finish with a
+    // single quote (") character.
+    debug_assert!(us.starts_with('"') && us.ends_with('"'));
+    let mut s = String::new();
+    // We iterate over all characters except the opening and closing quote characters.
+    let mut i = '"'.len_utf8();
+    while i < us.len() - '"'.len_utf8() {
+        let c = us[i..].chars().nth(0).unwrap();
```

Comment:
Do I see it right that `nth(0)` is the same as `next()` here?

Reply:
Yes I guess they are. I don't really prefer one over the other.

Reply:
I think next is about 1 byte more succinct :)

Again, I won't get hung up on this.

Reply:
I must admit, I wish there was a way that didn't require an `unwrap`. I have a marginal preference for `nth(0)` simply because it looks a bit more like `[0]` which is what we'd really like to write.

Reply:
It turns out that if you do `"blah"[0]` the compiler suggests that you use `chars().nth(0)` so I think we might as well stick with what we've got.

[I understand why they removed `char_at`, but I wish they'd kept it in!]